packages/craftcms-vue/admintable/App.vue

```diff
@@ -23,19 +23,24 @@
           </admin-table-action-button>
         </div>
 
-        <div
-          v-if="search && !tableData.length"
-          class="flex-grow texticon search icon clearable"
-        >
+        <div v-if="search" class="flex-grow texticon search icon clearable">
+          <span class="texticon-icon search icon" aria-hidden="true"></span>
           <input
             class="text fullwidth"
             type="text"
             autocomplete="off"
             :placeholder="searchPlaceholderText"
-            v-model="searchTerm"
-            @input="handleSearch"
```

Comment:
@i-just what was the reason behind these lines needing to change? the `v-model` attribute  and `input` event should handle everything you need so you don't have to pass the value manually.

Reply:
As per our chat: 
- I have synced all properties between `packages/craftcms-vue/admintable/App.vue` and `src/web/assets/admintable/src/main.js` (and double-checked that they’re all in the readme file), and I’ve put them all in the alphabetical order; also added a comment to the `App.vue` so that it’s clear that we need to do this (at least for now)
- I have switched back to `v-model` and `@input`
- in terms of `rows` vs `this.$refs.vuetable.tableData` - that’s proving to be a bit more complicated than I expected; not giving up, but I need to jump back to something else first